eisbuk-admin/public/locales/it/translation.json

```diff
@@ -11,6 +11,7 @@
     "NonReversible": "Questa azione non è reversibile",
     "Slot": "",
     "Book": "Prenota",
+    "Reserved": "Prenotato",
```

Comment:
Also, I prefer "Booked" here, instead of having "Book" and "Reserved" (for the differenct states of same action)